Tasks/KubectlInstallerV0/task.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "8413C881-4959-43D5-8840-B4EA0FFC5CFD",
+    "name": "KubectlInstaller",
+    "friendlyName": "Kubectl tool installer",
+    "description": "Install Kubectl on agent machine.",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=851275",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=851275)",
+    "category": "Tool",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "preview": true,
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 150,
```

Comment:
0.0.1

Reply:
minor is now the sprint version
    and patch is the change made during the sprint